dev/com.ibm.websphere.javaee.persistence.2.2/bnd.bnd

```diff
@@ -0,0 +1,45 @@
+#*******************************************************************************
+# Copyright (c) 2017 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Bundle-SymbolicName: com.ibm.websphere.javaee.persistence.2.2; singleton:=true
+#Bundle-Activator: com.ibm.ws.javaee.persistence.internal.HybridPersistenceActivator
```

Comment:
Is it intentional that the Bundle-Activator is commented out?

Reply:
Ah, you found a "TODO" that I forgot about.  No, that shouldn't be commented out.